### ISSUE TITLE
Please add the Plimsoll symbol (circle with a horizontal line through it)

### ISSUE THREAD

**Issue by LloydLS** [KaTeX/KaTeX#2200]
Hi!

Please consider adding the Plimsoll symbol (https://en.wikipedia.org/wiki/Plimsoll), as provided by the chemstyle package (called `\standardstate` in that package). `\barcirc` would also work instead.

`\ominus` is similar, but not exactly the same (the horizontal line extends beyond the circle in the Plimsoll symbol) 

Chemstyle package documentation (page 4 has the relevant section):
http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/chemstyle/chemstyle.pdf

Many thanks!

**Comment by ronkok** [comment]
@LloydLS  Please try this macro code on the KaTeX demo page. Let me know if it fits your needs. If it does, I'll write up a PR. Also, which name is preferable: `\standardstate` or `\barcirc`?

```
\gdef\barcirc{{\mathrlap{\mathchoice{\kern{0.145em}}{\kern{0.145em}}{\kern{0.1015em}}{\kern{0.1015em}}\circ}{-}}}

\barcirc

G^\barcirc \\
\int_{T^\barcirc}^{T_\text{out}}

```

**Comment by LloydLS** [comment]
Hi @ronkok 

That looks fantastic! 

I'd probably suggest the name `\barcirc`, just because sometimes a superscript circle (without a bar) is used to denote a standard state.

Thanks so much for this!

**Comment by edemaine** [comment]
The [comprehensive symbol list](http://tug.ctan.org/info/symbols/comprehensive/symbols-a4.pdf#page=216) mentions `\barcirc` from a `comp.text.tex` post, but no packages apparently call it that. (It also gives a macro definition for it, but it uses `\hfil` so we can't use it directly; @ronkok's definition seems to compose the same symbols, `\circ` and minus.) It does mention that `stmaryrd.sty` as well as `boisik.sty` define `\minuso` with a similar symbol, so `\minuso` is another reasonable choice for the name -- perhaps the best one?

**Comment by LloydLS** [comment]
Hi @edemaine 

I wasn't sure whether you were replying to me, so apologies for the delay. 

I'm not sure what the guidelines are for naming new symbols in KaTeX; presumably we want to be somewhat consistent with LaTeX? 

If we could name it anything we want, then maybe `\plimsoll` would be appropriate (hopefully wouldn't conflict with any nautical LaTeX users...)

Ultimately, anything you like to call it is fine by me.

Best regards!

**Comment by edemaine** [comment]
In general, we strive for consistency with LaTeX (not "somewhat consistency"). The intent is that someone can take code that runs in KaTeX and port it directly over to LaTeX without any changes (but possibly including a bunch of packages). Hence my suggestion to use `\minuso`. You can always define your own alias, just as you could in LaTeX.

**Comment by LloydLS** [comment]
Makes sense, `\minuso` sounds good to me!

**Comment by kevinbarabash** [comment]
Since `\minuso` composes `\circ` and `-` then it's possible we can add without adding it a font.

**Comment by ronkok** [comment]
>  it's possible we can add without adding it a font.

Yes, in HTML. But I have not yet thought of a good way to compose the two characters reliably in font-agnostic MathML. I'll go ahead and turn in a PR for the HTML, but we should be aware that this issue is not completely resolved.

**Comment by ronkok** [comment]
Well, I'm having submodule problems. I've tried to update my remote repository with `git merge upstream/master` and `git submodule update --init --recursive`  but in both cases I get a Git error message that reads:

```
error: Your local changes to the following files would be overwritten by checkout:
        src/fonts/generate_fonts.py
        src/fonts/lib/Space.ttx
        src/fonts/makeFF
Please commit your changes or stash them before you switch branches.
```

I have not made any local changes to these files and I don't know what to do next. Anybody have an idea that would help me?

**Comment by kevinbarabash** [comment]
@ronkok if you `cd` into `src/fonts` and run `git status`, what does it printout?

**Comment by ronkok** [comment]
> what does it printout?

This:

```
HEAD detached at 7deb3fd
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   ../../.circleci/config.yml
        modified:   ../../CONTRIBUTING.md
        modified:   ../../Dockerfile
        modified:   ../../LICENSE
        modified:   ../../README.md
        modified:   ../../buildFonts.sh
        modified:   ../../buildMetrics.sh
        modified:   ../../fontMetricsData.js
        modified:   ../../fonts.less
        modified:   ../../package.json
        modified:   ../Makefile
        modified:   ../default.cfg
        modified:   .gitignore
        modified:   Makefile
        modified:   generate_fonts.py
        modified:   lib/Space.ttx
        modified:   makeBlacker
        modified:   makeFF
        modified:   xbbold.mf
        modified:   ../metrics/.gitignore
        modified:   ../metrics/README.md
        modified:   ../metrics/extract_tfms.py
        modified:   ../metrics/extract_ttfs.py
        modified:   ../metrics/format_json.py
        modified:   ../metrics/mapping.pl
        modified:   ../metrics/parse_tfm.py
```

**Comment by kevinbarabash** [comment]
Doing `git checkout -- .` inside `src/fonts` that should discard the changes and you should be able to do `git submodule update` from the root `KaTeX` folder.

**Comment by ronkok** [comment]
Success. @kevinbarabash Thank you very much. Now I can get on with that macro.

**Comment by ronkok** [comment]
Fixed via PR #2213.
